### PR TITLE
Added set title to index page for init webspaces

### PR DESCRIPTION
**Pull request by wachterjohannes**
__Tasks:__

- [ ] gather feedback for my changes
- [x] added changelog line

__Informations:__

| Q             | A
| ------------- | ---
| Tests pass?   | yes
| Fixed tickets | none
| BC Breaks     | none
| Doc           | none

### CODE REVIEW

File: src/Sulu/Bundle/CoreBundle/Command/WebspacesInitCommand.php [hunk 1]
```diff
@@ -158,6 +159,11 @@ private function setBasicLocalizationProperties(
             $node->setProperty($this->properties->getName('nodeType'), Structure::NODE_TYPE_CONTENT);
         }
 
+        if (!$node->hasProperty($this->properties->getName('nodeType'))) {
```
Comment (danrot) [inline src/Sulu/Bundle/CoreBundle/Command/WebspacesInitCommand.php]: Shouldn't it check here for the title?

File: src/Sulu/Bundle/CoreBundle/Command/WebspacesInitCommand.php [hunk 2]
```diff
@@ -158,6 +159,11 @@ private function setBasicLocalizationProperties(
...
+            // set title
+            $node->setProperty($this->properties->getName('title'), 'Sulu CMF');
```
Comment (danrot) [inline src/Sulu/Bundle/CoreBundle/Command/WebspacesInitCommand.php]: Does this set the title for all localizations? And shouldn't we use something more general like `Homepage`?

File: src/Sulu/Bundle/CoreBundle/Command/WebspacesInitCommand.php (hunk 1 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/CoreBundle/Command/WebspacesInitCommand.php]: yes i fix it (=

File: src/Sulu/Bundle/CoreBundle/Command/WebspacesInitCommand.php (hunk 2 above)
Comment (wachterjohannes) [inline src/Sulu/Bundle/CoreBundle/Command/WebspacesInitCommand.php]: yes the title will be set for all localizations ... for me this title is not so important ... but if we want we could place a translation or we ask for each startpage localization? i wouldn´t use a locale dependend title therefor Sulu CMF seems to me a good middle way (= /cc @chirimoya @dantleech  what did you think?

Comment (danrot) [inline src/Sulu/Bundle/CoreBundle/Command/WebspacesInitCommand.php]: I guess Homepage is a word that can be used in every language ;-)

**Pull request merged**